x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/MachineLearningFlyout/index.tsx

```diff
@@ -66,6 +66,11 @@ export class MachineLearningFlyout extends Component<Props, State> {
       return;
     }
 
+    const errorDescription = error?.body?.message;
```

Comment:
@ogupte How did you get a response containing `body.message`?

I'm currently getting this error response:
```json
{
  "jobs": [
    {
      "id": "opbeans-node-request-high_mean_response_time",
      "success": false,
      "error": {
        "msg": "[status_exception] [1] result and [0] state documents exist for a prior job with Id [opbeans-node-request-high_mean_response_time]. Please create the job with a different Id",
        "path": "/_ml/anomaly_detectors/opbeans-node-request-high_mean_response_time",
        "query": {},
        "body": "{\"job_type\":\"anomaly_detector\",\"groups\":[\"apm\",\"opbeans-node\",\"request\"],\"description\":\"Detect anomalies in high mean of transaction duration\",\"analysis_config\":{\"bucket_span\":\"15m\",\"detectors\":[{\"detector_description\":\"high_mean(\\\"transaction.duration.us\\\")\",\"function\":\"high_mean\",\"field_name\":\"transaction.duration.us\"}],\"influencers\":[]},\"analysis_limits\":{\"model_memory_limit\":\"11MB\"},\"data_description\":{\"time_field\":\"@timestamp\"},\"model_plot_config\":{\"enabled\":true},\"custom_settings\":{\"created_by\":\"ml-module-apm-transaction\"}}",
        "statusCode": 409,
        "response": "{\"error\":{\"root_cause\":[{\"type\":\"status_exception\",\"reason\":\"[1] result and [0] state documents exist for a prior job with Id [opbeans-node-request-high_mean_response_time]. Please create the job with a different Id\"}],\"type\":\"status_exception\",\"reason\":\"[1] result and [0] state documents exist for a prior job with Id [opbeans-node-request-high_mean_response_time]. Please create the job with a different Id\"},\"status\":409}"
      }
    }
  ],
  "datafeeds": [
    {
      "id": "datafeed-opbeans-node-request-high_mean_response_time",
      "success": false,
      "started": false,
      "error": {
        "msg": "[resource_not_found_exception] No known job with id 'opbeans-node-request-high_mean_response_time'",
        "path": "/_ml/datafeeds/datafeed-opbeans-node-request-high_mean_response_time",
        "query": {},
        "body": "{\"job_id\":\"opbeans-node-request-high_mean_response_time\",\"indices\":[\"apm-*\"],\"query\":{\"bool\":{\"filter\":[{\"term\":{\"service.name\":\"opbeans-node\"}},{\"term\":{\"processor.event\":\"transaction\"}},{\"term\":{\"transaction.type\":\"request\"}}]}}}",
        "statusCode": 404,
        "response": "{\"error\":{\"root_cause\":[{\"type\":\"resource_not_found_exception\",\"reason\":\"No known job with id 'opbeans-node-request-high_mean_response_time'\"}],\"type\":\"resource_not_found_exception\",\"reason\":\"No known job with id 'opbeans-node-request-high_mean_response_time'\"},\"status\":404}"
      }
    }
  ],
  "kibana": {}
}
```

Reply:
To be clear, I get that error message when there are no ML nodes available:

![image](https://user-images.githubusercontent.com/209966/82315075-0a64a400-99cb-11ea-873f-a648038a39c2.png)


I think we should investigate how our UI handles in this case, and show an appropriate error message. Right now I'm just seeing the usual message:

<img src="https://user-images.githubusercontent.com/209966/82315748-0c7b3280-99cc-11ea-8dc4-fbce7f164662.png" />

Reply:
![82315748-0c7b3280-99cc-11ea-8dc4-fbce7f164662](https://user-images.githubusercontent.com/4104278/82319317-a2658c00-99d1-11ea-8d59-e38933ef7b2c.png)


We should at the very least remove the sub headline `Creating ML job failed`. Toasts will have one title and message.